content/docs/user-guide/data-pipelines/defining-pipelines.md

```diff
@@ -0,0 +1,237 @@
+# Defining Pipelines
+
+DVC pipelines are written as collections of data processing <abbr>stages</abbr>
+in `dvc.yaml` files. These represent data workflows that you want to
```

Comment:
I think somwhere in this doc worth describing that we allow for multiple dvc.yamls ...

Reply:
Good call. Added a paragraph about this in the DAG section.